### PR TITLE
Fixes #9841: Error parsing inventory

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/9841

### CODE REVIEW

File: initial-promises/node-server/inventory/1.0/fusionAgent.cf [hunk 1]
```diff
@@ -452,27 +452,27 @@ bundle agent addInformationsToInventory
...
         comment       => "Adding server roles informations to inventory",
         edit_line     => add_server_roles_information_to_inventory("@{addInformationsToInventory.rudder_roles}"),
-        edit_defaults => def_no_backup;
+        edit_defaults => def_no_backup_size("25M");
 
     android::
       "${g.rudder_var_tmp}/inventory/.*.ocs"
         edit_line => add_information_to_android_inventory(${android_fullname}, ${android_kernelname}, ${android_kernelversion}, ${android_name}, ${android_version}),
-        edit_defaults => def_no_backup;
+        edit_defaults => def_no_backup_size("25M");
```
Comment (armeniaca) [inline initial-promises/node-server/inventory/1.0/fusionAgent.cf]: also for android?

Comment (armeniaca) [inline initial-promises/node-server/inventory/1.0/fusionAgent.cf]: also for android?

**Comment by amousset** [comment]
ping @ncharles

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**